spec/polymorphism_spec.cr

```diff
@@ -0,0 +1,64 @@
+require "./spec_helper.cr"
+
+private class CommentForm < Comment::SaveOperation
+  permit_columns commentable_id
+  permit_columns commentable_type
+
+  def prepare
+    # Make sure we only save values we understand
+    validate_inclusion_of commentable_type, in: ["Employee", "Company"]
```

Comment:
This is great, but I believe should be done automatically when specifying in the model that it is polymorphic with `polymorphic_belongs_to` as described in #145

Reply:
tackling this next!

Reply:
couldn't figure out how to add this validation on the default operation, could you please give me some direction?

Reply:
solved! 🎉